auth/auth_test.go

```diff
@@ -2431,49 +2430,6 @@ func TestRoleSoftDelete(t *testing.T) {
 	assert.Equal(t, expectedChannelHistory, role.ChannelHistory()["!"].Entries[0])
 }
 
-func TestRoleSoftDeleteCasMismatch(t *testing.T) {
```

Comment:
This test was removed as it tested the casUpdatePrincipal work done on a role and there is no longer one of these operations being performed for a role.